ui/imports/shared/views/AssetsView.qml

```diff
@@ -95,10 +95,10 @@ Control {
 
         proxyRoles: [
             // helper role for rendering section delegate
-            FastExpressionRole {
+            JoinRole {
                 name: "isCommunity"
-                expression: !!communityId ? "community" : ""
-                expectedRoles: ["communityId"]
+                roleNames: ["communityId"]
+                separator: ""
             },
```

Comment:
It won't work as expected because it will affect sort order.

I think that similar to our custom `UndefinedFilter` could work nicely here, but with slightly different logic, checking if not only for undefined but also "empty" values.

Reply:
Btw, is the slowness visible in the application?

Reply:
Yes, that's true :D Not the proper solution here. This is why I'm keeping it as a draft, so that I don't forget and try this again 😄 It's a low priority as the number of assets is currently quite limited, but something we'll need to fix eventually